aemedge/scripts/delayed.js

```diff
@@ -16,7 +16,7 @@ function getEnvType(hostname = window.location.hostname) {
 }
 
 function getScope(path = window.location.pathname) {
-  const fullScopePrefixes = ['/events/', '/podcasts/', '/research/', '/resources/', '/videos/'];
+  const fullScopePrefixes = ['/research/', '/resources/', '/videos/'];
```

Comment:
You still have `research`? 
Also some of today's Insight documents get sorted under /blogs/... should this be on the list as well?

Reply:
I follow the documentation and information I got from Angie Nink:
- in insights-mapped-final.xlsx (https://sap.sharepoint.com/:x:/r/sites/207899/Shared%20Documents/documentation/insights-mapped-final.xlsx?d=w581e1e2426d145bd848cb125d7005c0f&csf=1&web=1&e=aY9jY9) > Mapping Key > Content Folder: /research is a content folder path I have found
- /blogs content: I read Angie's reply here: https://adobe-dx-support.slack.com/archives/C06H8UNQ3RS/p1711455394198709?thread_ts=1711443252.163829&cid=C06H8UNQ3RS that all content that moves to /blogs will be counted as News content

Therefore, I assume, /research, /resource and /videos are valid insights paths